src/EnergyPlus/DataEnvironment.cc

```diff
@@ -181,6 +181,122 @@ namespace DataEnvironment {
 
 	// Functions
 
+	// Clears the global data in DataEnvironment.
+	// Needed for unit tests, should not be normally called.
+	void
+	clear_state()
```

Comment:
I'm curious what is the analog if this if the namespaces were evolved into classes?  Would you just destroy the object and recreate a new one.  Or would you still have this method exposed publicly?